pkg/repo/fetch.go

```diff
@@ -234,6 +237,41 @@ func fileGet(filename string) (*http.Response, error) {
 	return resp, nil
 }
 
+var netrcCached *netrc.Netrc
+
+func getNetrc() (*netrc.Netrc, error) {
+	if netrcCached != nil {
+		return netrcCached, nil
+	}
+	usr, err := user.Current()
+	if err != nil {
+		return nil, fmt.Errorf("getting current user: %w", err)
+	}
+	n, err := netrc.Parse(filepath.Join(usr.HomeDir, ".netrc"))
+	if err == nil {
+		netrcCached = n
+	}
+	return n, err
```

Comment:
I don't think you should return the err in this case, or maybe you should check for the file existing first?

Reply:
I'd expect there to be a parse error if the file is malformed. Are you suggesting we ignore that instead?

Reply:
oh, nevermind, I looked at the implementation and the only error it returns is if the file doesn't exist

Reply:
If the file is not there, which is most likely the case, shouldn't make the tool fail